fee-monitor/src/DynamicChecker.ts

```diff
@@ -277,6 +292,28 @@ export class DynamicChecker {
                     bytes: Buffer;
                 }
                 const { handlerId, bytes }: CustomBody = tx as any;
+                if (handlerId.eq(2)) {
+                    const decoded = RLP.decode(bytes as RLP.Input);
+                    if (decoded instanceof Array) {
+                        const [tag, ...rest] = decoded;
+                        const tagNum = tag.readUInt8(0);
+                        switch (tagNum) {
+                            case STAKE_CONSTANT.ACTION_TAG_SELF_NOMINATE: {
```

Comment:
It should withdraw CCC

Reply:
`checkBlockStatic` should handle self-nomination too. Any eligible account can self-nominate before the first term close. Pass `dynamicChecker.nominationDeposits` to `checkBlockStatic` and add the deposit to it.

Reply:
Ok, I checked the patch code. It look fine.